spec/test_after_commit_spec.rb

```diff
@@ -173,6 +173,31 @@ def rails42?
     end
   end
 
+  context "block behavior" do
+    it "does not fire if turned off" do
```

Comment:
`after { TestAfterCommit.enabled_by_default = true }` to make sure it's cleaned up all the time